openml/_api_calls.py

```diff
@@ -175,10 +175,13 @@ def _send_request(
     request_method, url, data, files=None,
 ):
     n_retries = config.connection_n_retries
+    max_retries = 10
```

Comment:
Is there a reason you changed this here and not in the config file? Also, what's the intuition of moving from a for loop to the while loop?

Reply:
Yes, I should add it to config instead, that's true. I'll do so.

Wrt replacing the for-loop with a while-loop, I am now incrementing the number of retries by 1 every time a database connection error happens. Since `n_retries` determine the iterations, I think the for-loop iterator is not mutable from within the loop. A while-loop allowed me to do that easily and therefore I felt it was more readable.